tests/test_schemas.py

```diff
@@ -616,8 +616,82 @@ def test_get_node_range(self):
     def test_get_node_required(self):
         return
 
-    def test_get_node_validation_rules(self):
-        return
+    @pytest.mark.parametrize(
+        "data_model", list(DATA_MODEL_DICT.keys()), ids=list(DATA_MODEL_DICT.values())
+    )
+    @pytest.mark.parametrize(
+        "node_label, node_display_name, expected_validation_rule",
+        [
+            # Test case 1: node label is provided
+            (
+                "PatientID",
+                None,
+                {"Biospecimen": "unique error", "Patient": "unique warning"},
+            ),
+            (
+                "CheckRegexListStrict",
+                None,
+                ["list strict", "regex match [a-f]"],
+            ),
+            # Test case 2: node label is not provided and display label is not part of the schema
+            (
+                None,
+                "invalid display label",
+                [],
+            ),
+            # Test case 3: node label is not provided but a valid display label is provided
+            (
+                None,
+                "Patient ID",
+                {"Biospecimen": "unique error", "Patient": "unique warning"},
+            ),
+        ],
+    )
+    def test_get_node_validation_rules_valid(
+        self,
+        helpers,
+        data_model,
+        node_label,
+        node_display_name,
+        expected_validation_rule,
```

Comment:
nit: typing for arguments

Reply:
Thanks. Just finished adding type hinting.